From 4d3e28de30d83d243ad8f4a340be9ac027f7ba79 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 1 Feb 2012 20:03:53 +0000
Subject: [PATCH] make: support for tar without -J option

Not all tar implemntation has support for -J yet. For example
busybox tar.

This trivial fix makes the build work out of the box on those systems.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 1de4b27..e88a243 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -315,7 +315,7 @@ src_test_udev_CPPFLAGS = $(udev_common_CPPFLAGS)
 
 # packed sysfs test tree
 test/sys/kernel/uevent_seqnum:
-	$(AM_V_GEN)mkdir -p test && tar -C test/ -xJf $(top_srcdir)/test/sys.tar.xz
+	$(AM_V_GEN)mkdir -p test && unxz -c $(top_srcdir)/test/sys.tar.xz | tar -C test/ -x
 
 BUILT_SOURCES += test/sys/kernel/uevent_seqnum
 
-- 
1.7.9

